tedge/src/cli/connect/common_mosquitto_config.rs

```diff
@@ -40,6 +40,11 @@ impl CommonMosquittoConfig {
 
         Ok(())
     }
+
+    pub fn with_port(self, port: u16) -> Self {
+        let listener = port.to_string() + " localhost";
+        Self { listener, ..self }
+    }
```

Comment:
How about some unit test for the method?

Reply:
added